update flows samples to use v1 API (#2633)
knative/docs: docs/eventing/samples/sequence/sequence-with-broker-trigger/README.md

> We are going to create the following logical configuration. We create a

Reviewer: Please fix all the places where this doc says "we" - https://developers.google.com/style/person

Author: This PR is only about updating the samples to use the V1 API. No additional text has been written. Can we merge this PR before the 0.16 doc release?

> PingSource, feeding events into the Broker, then we create a `Filter` that wires
> those events into a [`Sequence`](../../../flows/sequence.md) consisting of 3
> steps. Then we take the end of the Sequence and feed newly minted events back
> into the Broker and create another Trigger which will then display those events.

Reviewer: ```suggestion
PingSource, that feeds events to the broker, then create a filter that wires
those events into a [sequence](../../../flows/sequence.md) consisting of 3
steps. Then take the end of the sequence, and feed the newly created events back
into the broker, and create another trigger which will then display those events.
```

Reviewer: This is a very long and confusing list to be written this way. It would be better as a numbered list of steps IMHO.

## Prerequisites

For this example, we'll assume you have set up an `InMemoryChannel` as well as

> Knative Serving (for our functions). The examples use `default` namespace,

Reviewer 2: ```suggestion
Knative Serving (for our functions). The examples use the `default` namespace,
```
namespace is used twice in this sentence, but it's capitalized in one spot and lowercase in another, not sure which one should be used.

Reviewer: lowercase please

> For this example, we'll assume you have set up an `InMemoryChannel` as well as
> Knative Serving (for our functions). The examples use `default` namespace,

Reviewer: ```suggestion
- Knative Serving
- `InMemoryChannel`

**NOTE:** The examples use the `default` namespace.
```

Author: done

> again, if your broker lives in another Namespace, you will need to modify the
> examples to reflect this.

Reviewer: ```suggestion
If your broker lives in another namespace, you must modify the
examples to reflect this.
```
